processing/src/main/java/io/druid/query/search/search/DefaultSearchQueryMetrics.java

```diff
@@ -0,0 +1,238 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.query.search.search;
```

Comment:
Double `search.search` package

Reply:
search pkg contains another `search` pkg, https://github.com/druid-io/druid/tree/master/processing/src/main/java/io/druid/query/search/search.

Reply:
Indeed, but seems like a error? I see no reason for that

Reply:
Yes, seems like an error. I will create a pr either to move all the classes from `search.search` package to `search` package or rename `search.search` pkg to something else ?

Reply:
IMO just move from `search.search` to `search`

Reply:
👍